corso_website_springboot/src/test/java/com/corso/springboot/selenium_tests/ViewServiceDetailsTest.java

```diff
@@ -0,0 +1,58 @@
+package com.corso.springboot.selenium_tests;
+
+import com.codeborne.selenide.SelenideElement;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static com.codeborne.selenide.Selenide.*;
+import static com.codeborne.selenide.Condition.visible;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
+public class ViewServiceDetailsTest {
```

Comment:
Did you exclude this test for the ci/cd

Reply:
nope i forgot by accident, just pushed it!